web-console/knikubevirt/openshift-vms-overview/openshift-vms-overview.md

```diff
@@ -0,0 +1,25 @@
+# OpenShift VMs Overview
+
+The purpose of this PR is to explore what Virtual Machines could look like in the Overview and Resources sections of Openshift/CNV.
+This is the first pass so really just trying to understand exactly what needs to be surfaced there.
+
+Through the Overview page the user can see the VM’s based on the application it is associated with via it’s label and and also see some sparkline charts that illustrate the usage of Network, CPU and Memory for the past 7 days.
+
+![sparkline](Openshift-overview-virtual-machine-sparkline.png)
+
+Clicking on the Kebab would offer a list of options depending on that state of the VM.
+
+![dropdown menu options](Openshift-overview-virtual-machine-dropdown.png)
```

Comment:
Looking at the latest code, I think there are still a few small updates we should make...

1) OpenShift isn't using the small donut to represent the pod count currently. Maybe @beanh66 would have some insight into whether we should keep this or not? 
2) It might be worth adding in the metrics to the row for container items. They are missing in the designs here.

Here's a screenshot of the latest incase it's helpful:
<img width="1417" alt="workloads" src="https://user-images.githubusercontent.com/208455/60918896-636b9380-a262-11e9-928b-5b555bcf6413.png">

Reply:
Thanks for the comments @lizsurette 
Files are updated.